dcrtimed/log.go

```diff
@@ -40,14 +40,16 @@ var (
 	// application shutdown.
 	logRotator *rotator.Rotator
 
-	log     = backendLog.Logger("DCRT")
-	fsbeLog = backendLog.Logger("FSBE")
+	log       = backendLog.Logger("DCRT")
+	fsbeLog   = backendLog.Logger("FSBE")
+	walletLog = backendLog.Logger("DCRW")
```

Comment:
I don't understand what you're trying to do here.  dcrwallet runs as a completely separate process.  You should not be logging messages from dcrwallet in dcrtime.

Reply:
Makes sense, as the wallet runs separately and has it's own logs